frontend/components/TableContainer/DataTable/TooltipTruncatedTextCell/TooltipTruncatedTextCell.tsx

```diff
@@ -8,6 +8,9 @@ import { COLORS } from "styles/var/colors";
 
 interface ITooltipTruncatedTextCellProps {
   value: string | number | boolean;
+  /** Tooltip to dispay. If this is provided then this will be rendered as the tooltip content. If
+   * not the value will be displayed as the tooltip content. Defaults to `undefined` */
+  tooltip?: React.ReactNode;
```

Comment:
do we manipulate `value` in some way assuming it's a `string | number | boolean`? otherwise (and just a suggestion) we could change `value` to be `React.ReactNode`? so we don't have two props doing similar things

```suggestion
  value: React.ReactNode;
```

Reply:
makes sense. Updating to React.ReactNode